### DISCUSSION TITLE
Ideas Para el Formulario de Postulate

### DISCUSSION THREAD

**Discussion by MGeovany** [MGeovany/cei-web-portal#31] [Q&A] [answered]
El Formulario aun no tiene un diseño completo, necesita mas colores o un rediseño mas basico

**Comment by wpayze** [comment, marked as answer]
Mira aqui te dejo una idea

![](https://quish.tv/img/blog/66/bulma-registration-form-w3hubs.png)

Solo hay que ponerle los colores de la U

**Reply by wpayze** [reply]
pero de ahi esta chequesito mi rey
